.github/workflows/release_constructor.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Installer
+on: 
+  push:
+    tags:
+    - '*'
+jobs:
+  build:
+    defaults:
+      run:
+        shell: ${{ matrix.shell }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        include:
+          - os: ubuntu-latest
+            shell: bash -l {0}
+          # - os: windows
+          #   shell: cmd /C call {0}
+          # - os: macos
+          #   shell: bash -l {0}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          environment-file: environment.yml
+          python-version: ${{ matrix.python-version }}
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          mamba-version: "*"
+          use-mamba: true
+      - name: Check Python version
+        run: python --version
+      - name: constructor 
+        run: constructor --output-dir dist/main
+      - name: Rename installer
+        run: mv dist/main/*.sh dist/main/${{ matrix.python-version }}-${{ matrix.os }}.sh
+      - name: Upload Installer
+        uses: actions/upload-artifact@v2
+        with:
+          name: installer-${{ matrix.python-version }}-${{ matrix.os }}
+          path: dist/main/**/*
+      # - uses: softprops/action-gh-release@v1
+      #   with:
+      #     draft: true
+      #     files: dist/main/*
+      #   env:
+      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is  
+  release:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download all artifacts
+        uses: actions/download-artifact@v2
+      # - name: Create Draft Release
+      #   uses: softprops/action-gh-release@v1
+      #   with:
+      #     draft: true
+      #     files: ./**/*
+      #   env:
+      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create Release
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: "**/*"  # Replace with your actual artifacts path
```

Comment:
@ctr26 this is the version updater but seems to be empty. At the moment there are different versions for notebooks, docker configuration files and I guess that for the constructor as well. How is this one meant to be used?

Reply:
It tags and names the builds based on the python version (at the moment) it could be used down the line for a constructor build per notebook maybe

Reply:
ok, this is because the constructor for sure will change the version depending on the supported python version, no? We have different python versions for different notebooks (I will open an issue about this). Will this affect the constructor?